docs/maui/behaviors/icon-tint-color-behavior.md

```diff
@@ -0,0 +1,88 @@
+---
+title: IconTintColorBehavior - .NET MAUI Community Toolkit
+author: vhugogarcia
+description: "The IconTintColorBehavior is a behavior allows you to tint an image."
+ms.date: 06/27/2022
+---
+
```

Comment:
Do not merge until we merge and release: 

https://github.com/CommunityToolkit/Maui/pull/456